testdrive/testdrive.php

```diff
@@ -54,10 +56,10 @@ function testdrive_register_account($a,$b) {
 function testdrive_cron($a,$b) {
```

Comment:
Parameter `$b` is unused and can be removed.

Reply:
How about adding `App` here?

Reply:
Sure, I focused on the mistakes, not on the missing. 😄